web/src/main/java/org/springframework/security/web/util/matcher/RequestMatchers.java

```diff
@@ -60,7 +83,236 @@ public static RequestMatcher not(RequestMatcher matcher) {
 		return (request) -> !matcher.matches(request);
 	}
 
+	/**
+	 * Create {@link RequestMatcher}s whose URIs do not have a servlet path prefix
+	 * <p>
+	 * When there is no context path, then these URIs are effectively absolute.
+	 * @return a {@link Builder} that treats URIs as relative to the context path, if any
+	 * @since 6.5
+	 */
+	public static Builder request() {
+		return new Builder();
+	}
+
+	/**
+	 * Create {@link RequestMatcher}s whose URIs are relative to the given
+	 * {@code servletPath}.
+	 *
+	 * <p>
+	 * The {@code servletPath} must correlate to a configured servlet in your application.
```

Comment:
This sounds like `servletPath` might be the equivalent of `<servlet-mapping>/<url-pattern>`, but I suspect it is corresponding to `HttpServletRequest.servletPath`. Perhaps, this could be clarified?